### PR TITLE
#67301 dashboard implement domain model

### PR DESCRIPTION
**Pull request by Klaas-Ritense**

### CODE REVIEW

File: dashboard/src/main/kotlin/com/ritense/dashboard/domain/Dashboard.kt [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+@Entity
+@Table(name = "dashboard")
+data class Dashboard(
+
+    @Id
+    @Column(name = "id", updatable = false, nullable = false, unique = true)
+    val id: UUID = UUID.randomUUID(),
+
+    @Column(name = "title", updatable = false, nullable = false)
+    val title: String,
```
Comment (theo-ritense) [inline dashboard/src/main/kotlin/com/ritense/dashboard/domain/Dashboard.kt:34]: Did we consider how/if we want to translate these titles?

File: dashboard/src/main/kotlin/com/ritense/dashboard/domain/WidgetConfiguration.kt [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+import javax.persistence.Column
+import javax.persistence.Entity
+import javax.persistence.FetchType
+import javax.persistence.Id
+import javax.persistence.JoinColumn
+import javax.persistence.ManyToOne
+import javax.persistence.Table
+
+@Entity
+@Table(name = "widget_configuration")
```
[the remark this answers is not in the document]
Comment (theo-ritense) [reply dashboard/src/main/kotlin/com/ritense/dashboard/domain/WidgetConfiguration.kt:31]: I agree.

Comment (theo-ritense) [inline dashboard/src/main/kotlin/com/ritense/dashboard/domain/WidgetConfiguration.kt:31]:
```suggestion
@Table(name = "dashboard_widget_configuration")
```

File: dashboard/src/main/resources/config/liquibase/changelog/20230531-add-dashboard-table.xml [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+                <constraints nullable="false"/>
+            </column>
+            <column name="sort_order" type="${intType}">
+                <constraints nullable="false"/>
+            </column>
+        </createTable>
+    </changeSet>
+
+    <changeSet author="Ritense" id="2">
+        <createTable tableName="widget_configuration">
```
Comment (theo-ritense) [inline dashboard/src/main/resources/config/liquibase/changelog/20230531-add-dashboard-table.xml:36]:
```suggestion
        <createTable tableName="dashboard_widget_configuration">
```

File: dashboard/src/main/resources/config/liquibase/changelog/20230531-add-dashboard-table.xml [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+                <constraints nullable="false"/>
+            </column>
+            <column name="sort_order" type="${intType}">
+                <constraints nullable="false"/>
+            </column>
+        </createTable>
+    </changeSet>
+
+    <changeSet id="3" author="Ritense">
+        <addForeignKeyConstraint baseTableName="widget_configuration"
```
Comment (theo-ritense) [inline dashboard/src/main/resources/config/liquibase/changelog/20230531-add-dashboard-table.xml:59]:
```suggestion
        <addForeignKeyConstraint baseTableName="dashboard_widget_configuration"
```

File: dashboard/src/main/kotlin/com/ritense/dashboard/domain/WidgetConfiguration.kt (hunk 2 above)
[the remark this answers is not in the document]
Comment (Klaas-Ritense) [reply dashboard/src/main/kotlin/com/ritense/dashboard/domain/WidgetConfiguration.kt]: changed

**Review by theo-ritense — APPROVED** [review]

**Pull request merged**